### DISCUSSION TITLE
SecurityOnion-idh install

### DISCUSSION THREAD

**Discussion by wangfei906** [Security-Onion-Solutions/securityonion#8672] [Unsupported Versions]
I got an error when I installed it, did I install it successfully?

![image](https://user-images.githubusercontent.com/37692493/188771719-d950b623-5521-41a5-948b-81de1ab89d26.png)
![image](https://user-images.githubusercontent.com/37692493/188771766-12a05f63-bc06-4efa-8305-3504785e2772.png)

**Comment by reyesj2** [comment]
It appears this error happened before attempting to add it to any existing distributed environment, correct?
Have you verified your ISO following the [instructions](https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md) ?
